### ISSUE TITLE
Are there pretrained models available?

### ISSUE THREAD

**Issue by Maestro157** [tensorlayer/hyperpose#201]
I found only a pretrained model for hao28_experimental in this repository: https://github.com/tensorlayer/pretrained-models.

Are there anywhere pretrained models for the vgg, vggtiny and mobilenet available?

**Comment by bela127** [comment]
Hi, i have been working with the openpose-plus model for a while now, and i did not find other pretrained models, except the ones you mentioned.

I trained a model from scratch, but have a similar issue as in:
#199

well its trained a lot longer, and converged a litle more...
if i can solve the issue with the jumping loss i will publish the finished model.
if it can be solved...

for now i can only advice transfer learning of hao28_experimental

**Comment by bela127** [comment]
another solution would be to load the weights of only the mobilenet/vgg classifikator into the open-pose net, so you only have to train the last layers (stages) of your model.

you could just use the mobilenet/vgg weights here:
https://github.com/tensorlayer/pretrained-models/tree/master/models

I cloned and changed the code a bit, so this can be done easily with the training config
https://github.com/bela127/openpose-plus/blob/master/train.py

**Comment by ccl-private** [comment]
Hi, @bela127. How is it going? I am going to try the method that you mentioned above. (downloading datasets~)
If convenient, please show some results on the test set. Really appreciate.

**Comment by bela127** [comment]
Hi, sorry i was busy lately
I tried a lot of different approaches with no result.

I lastly switched to a totally different implementation of the model.
this works perfect so no need for open-pose-plus

the other implementation:
https://github.com/ildoonet/tf-pose-estimation

or my fork with some additions:
https://github.com/bela127/tf-pose-estimation
